content/jest.globalTeardown.ts

```diff
@@ -1,5 +1,54 @@
+import chalk from 'chalk'
+import columnify from 'columnify'
+import fs from 'fs'
+import path from 'path'
+import jestConfig from './jest.config'
+import { ApiCoverage } from './jest.globalSetup'
+
 const globalTeardown = async (): Promise<void> => {
+  if (process.env.API_COVERAGE === 'true') {
+    await printApiCoverage()
+  }
+
   await globalThis.__POSTGRES_CONTAINER__?.stop()
 }
 
+type ApiCoverageRow = {
+  route: string,
+  method: string,
+  statuses: string[]
+}
+
+async function printApiCoverage() {
+  const apiCoveragePath = path.join(__dirname, jestConfig.coverageDirectory, 'api-coverage.json')
+  const coverage: ApiCoverage = JSON.parse(((await fs.promises.readFile(apiCoveragePath)).toString()))
```

Comment:
Does this work with tests with concurrency>1?

Reply:
Unfortunately not with this approach. I can try doing it in another PR, making it work in parallel in the CI